### ISSUE TITLE
Brisk Menu crashes when select "Edit Menus" with menulibre or mozo installed

### ISSUE THREAD

**Issue by ThanosApostolou** [solus-project/brisk-menu#76]
Original bug reported on Solus: https://dev.solus-project.com/T5094 with messages from `journalctl -p4 -xe`:
```
Nov 21 17:03:00 thanos-pc org.mate.panel.applet.BriskMenuFactory[939]: /usr/lib/python3.5/site-packages/menulibre/MenuEditor.py:28: PyGIWarning: GMenu was imported without specifying a version first. Use gi.require_version('GMenu', '3.0') before import to ensure that the right version gets loaded.
Nov 21 17:03:00 thanos-pc org.mate.panel.applet.BriskMenuFactory[939]:   from gi.repository import GdkPixbuf, Gio, GLib, GMenu, Gtk
Nov 21 17:03:00 thanos-pc org.mate.panel.applet.BriskMenuFactory[939]: No menu prefix found, MenuLibre will not function properly.
Nov 21 17:03:00 thanos-pc org.mate.panel.applet.BriskMenuFactory[939]: WARNING:menulibre: getDefaultMenuPrefix() 'No menu prefix found, MenuLibre will not function properly.'
Nov 21 17:03:01 thanos-pc org.mate.panel.applet.BriskMenuFactory[939]: WARNING:menulibre: getDefaultMenuPrefix() 'No menu prefix found, MenuLibre will not function properly.'
Nov 21 17:03:01 thanos-pc org.mate.panel.applet.BriskMenuFactory[939]: WARNING:menulibre: getDefaultMenuPrefix() 'No menu prefix found, MenuLibre will not function properly.'
```

However, I managed to reproduce on Manjaro too with brisk-menu 0.5.0 (where 0.4.5 works fine).

**Comment by ThanosApostolou** [comment]
To help you more, I built brisk-menu from git from various commits, using this package.yml: https://pastebin.com/cjbjFLgU and I found which commit causes the problem.

It works properly until commit: https://github.com/solus-project/brisk-menu/commit/3f9fff3336efbc98bc1887e5354852d04ed4f9c2

It has the mentioned bug from commit: https://github.com/solus-project/brisk-menu/commit/b2af6872d3f1598869bd66a35551548df1939206 and later.

**Comment by ikeydoherty** [comment]
3f9ff3 was reverted and unrelated, it was in-tree to help me debug

**Comment by ThanosApostolou** [comment]
yeah the b2af687 is the important one, all the previous commit don't matter.

**Comment by notramo** [comment]
Only the panel button disappears when the menu editing is launched. Pressing the Mod4 button launches the menu correctly even when the panel button is invisible (crashed).

**Comment by Yaksinikos** [comment]
#### Expected behaviour
Open customization without disappearing / hiding icon on panel.

#### Actual behaviour
Menu-Item/Icon/Entry disappears or hides.
But is further accessible by super-key.
It is not accessible by right-clicking on its actual position.

#### Steps to reproduce the behaviour
1. Right-click on Brisk-Menu-Starter.
2. Select customization.

#### DE general version
Mate: 1.20.4

#### Package version
Brisk Menu Starter: 0.5.0
Mate-Panel: 1.20.5

#### Linux Distribution
Manjaro Linux 64bit
Kernel: 4.19.30-1-Manjaro x86_64
